packages/manager/modules/cloud-connect/src/details/overview/add-pop-configuration/index.js

```diff
@@ -29,6 +29,7 @@ angular
         },
       });
     },
-  );
+  )
+  .run(/* @ngTranslationsInject:json ./translations */);
```

Comment:
If we do that it means that we lazy load only the component and not all things related to it.

Reply:
Maybe what's needed is a `$translate.refresh` before entering the states

Reply:
I'll try that

Reply:
Yes but i think the refresh might be done in the upper state, i actually added it and it works now

Reply:
@cbourgois as it is asynchronous, component might be loaded and translations still being fetched right ? 
Maybe we can prevent this error more globally with scoping to the root state ?

Reply:
But some translations (that are in the same translations file) are OK, some not.
I think it's because the modal is displayed before the translations's injections is completed, and 
- when using `translate` directive, it's OK, 
- when using `translation` filter in one time expression (`::`), it doesn't works.

I try to understand adding the following code in the template:

```html
<p data-translate="cloud_connect_pop_configure_title"></p>
<p>{{:: 'cloud_connect_pop_configure_title' | translate }}</p>
<p>{{ 'cloud_connect_pop_configure_title' | translate }}</p>
```

it displays

```html
Ajout d’une configuration PoP
cloud_connect_pop_configure_title
Ajout d’une configuration PoP
```

So I try to remove one time expression to have

```html
<oui-modal
        data-heading="{{ 'cloud_connect_pop_configure_title' | translate }}"
```

instead of

```html
<oui-modal
        data-heading="{{:: 'cloud_connect_pop_configure_title' | translate }}"
```
Cf https://github.com/ovh/manager/blob/master/packages/manager/modules/cloud-connect/src/details/overview/add-pop-configuration/template.html#L8

But it doesn't work because the first time the modal is rendered I think translation are not injected, and as oui-modal#heading is a one time binding (cf https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-modal-introduction--page), it won't be updated when translations are injected.

Adding a `$translate.refresh()` could help to fix the bug, but it doesn't fix the root cause.

Reply:
We will fix the root cause in another PR. This one fixes this issue, so no blocker.